ffmpeg_plugin/0001-lavc-svt_hevc-add-libsvt-hevc-encoder-wrapper.patch

```diff
@@ -245,6 +245,9 @@ index 0000000..d9ac04c
 +    }
 +
 +    param->targetBitRate = avctx->bit_rate;
++    param->vbvMaxrate = avctx->rc_max_rate;
++    param->vbvBufsize = avctx->rc_buffer_size;
++    param->vbvBufInit = 90;
```

Comment:
Hi @doublej472 , per @jsunintel 's feedback, please define it as a macro and mention the derived default [value](https://github.com/OpenVisualCloud/SVT-HEVC/blob/master/Source/API/EbApi.h#L524). Thanks!